Game/Source/ThirdPersonShooter/BigGym/BigGymCharacter.cpp

```diff
@@ -0,0 +1,391 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+#pragma optimize("", off)
```

Comment:
Would probably remove this (and the corresponding line at the bottom of the file) from the checked-in code. Sorry, forgot I checked this in